doc/src/contributing/new-contributors-guide/dev-setup.md

```diff
@@ -184,7 +184,7 @@ If you prefer to use `pip` and `venv`, you can use something like
 cd sympy
 python -m venv .venv
 source .venv/bin/activate
-pip install -e .
+pip install sympy["dev"]
```

Comment:
I think this will install from pypi.

Maybe `pip install -e .[dev]`?

https://stackoverflow.com/questions/30239152/specify-extras-require-with-pip-install-e

We should make sure the syntax works in both bash and cmd if possible.